Programmable FIR Filter

JustinWilliamsMSU opened the pull request:
Final testing folder of the programmable FIR DE10 Project.

fe-tdavis committed:
Initial FFT-in-fabric commit

Jevitate committed:
compiled dma, signal tap start

Jevitate committed:
spi stuff

Jevitate committed:
working PFIR model

Jevitate committed:
Add files via upload

Jevitate committed:
Add files via upload

Jevitate committed:
Add files via upload

Jevitate committed:
Add files via upload

Jevitate committed:
fixed quartus project with bigger address width

Jevitate committed:
merge

Jevitate committed:
update qsys file"

JustinWilliamsMSU committed:
Adding pFIR project compiled on my machine.

JustinWilliamsMSU committed:
Merge branch 'static_fir_test' of https://github.com/fpga-open-speech-tools/de10nano_projects into static_fir_test

Jevitate committed:
updated Avalon wrapper

Jevitate committed:
Merge branch 'static_fir_test' of https://github.com/fpga-open-speech-tools/de10nano_projects into static_fir_test

Jevitate committed:
adding output file

Jevitate committed:
adding tcl scripts

Jevitate committed:
hopefully correct soc_sopcinfo

JustinWilliamsMSU committed:
Updated TCL scripts and avalon wrapper vhd file with most current version.

Jevitate committed:
working PFIR Quartus Project

Jevitate committed:
Merge branch 'static_fir_test' of https://github.com/fpga-open-speech-tools/de10nano_projects into static_fir_test

Jevitate committed:
hps_isw_handoff folder

Jevitate committed:
output folder

tvannoy reviewed (commented):
What are the DMA and LXDE projects for? It looks like the LXDE one was for the scrolling spectrogram @fe-tdavis @Jevitate. ﻿

tvannoy committed:
revert passthrough project back to version in dev

tvannoy committed:
revert simple gain to version in dev

tvannoy committed:
Changes to make the project compile

The project didn't compile out of the box, probably because some
manual changes didn't get pushed earlier.

tvannoy committed:
Add pFIR-related device driver, device trees, and shell scripts

These previously lived in component_library. I'm putting them here for
now since they are needed to make the pFIR project work.

This project will likely get deleted in the future once we fully integrate
support for dual port RAM into the FOST system.

tvannoy committed:
add readme

tvannoy commented:
Project has been tested and working. Light documentation on how to run the project has been added to a README. 

This project is unlikely to live on in it's current form, but it will server as a reference for our future development and integration of fabric-based DPRAM.

tvannoy committed:
Merge branch 'dev' into static_fir_test
